.github/workflows/ci.yml

```diff
@@ -25,10 +25,32 @@ jobs:
           npm ci
           npm run build --if-present
           npm run test
+
+          # Test generic template
           cd templates/project-ts
           npm ci || true
           npm run test
           rm -rf node_modules
+          cd ../..
+
+          # Ensure example apps are working
```

Comment:
Do you know about how long do these tests take to run on CI? If it's slow, we could consider splitting them into parallel jobs since they can run independently.

Reply:
Long enough that it's probably worth it! I'll split them up.